### PR TITLE
kmodule: consider modules.builtin if present

### PR DESCRIPTION
**Pull request by ddiss**
Signed-off-by: David Disseldorp <ddiss@suse.de>

### CODE REVIEW

**Comment by rjoleary** [comment]
This looks good! Can you rebase on top of master?

File: pkg/kmodule/kmodule_linux.go [hunk 1]
```diff
@@ -124,6 +129,32 @@ func ProbeOptions(name, modParams string, opts ProbeOpts) error {
...
+	for scanner.Scan() {
+		txt := scanner.Text()
+		modPath := filepath.Join(moduleDir, strings.TrimSpace(txt))
+		if deps[modPath] == nil {
+			deps[modPath] = new(dependency)
+		}
+		deps[modPath].state = builtin
+	}
+
+	if err := scanner.Err(); err != nil {
```
Comment (hugelgupf) [inline pkg/kmodule/kmodule_linux.go]: Could be simplified to just 

```go
return scanner.Err()
```

Comment (ddiss) [reply pkg/kmodule/kmodule_linux.go]: Thanks for the feedback. I've pushed a new version with this change.

**Comment by rminnich** [comment]
Thanks!

**Pull request merged**